spec/feature/feature_spec.rb

```diff
@@ -0,0 +1,38 @@
+# require 'capybara/rspec'
+# require 'takeaway'
+#
+# describe 'Customer features' do
+#   subject(:takeaway) { Takeaway.new }
+#
+#   feature 'see name and prices of dishes on menu' do
+#     it 'includes menu item' do
+#       expect(takeaway.show_menu).to match('Lasagne Al Forno: £10.5')
```

Comment:
Style/AsciiComments: Use only ascii symbols in comments. (https://github.com/bbatsov/ruby-style-guide#english-comments)